tests/dictionary1.py

```diff
@@ -1,7 +1,7 @@
 def test_Dict():
     x: dict[i32, i32]
     x = {1: 2, 3: 4}
-    # x = {1: "2", "3": 4} -> sematic error
+    # x = {1: "2", "3": 4} -> semantic error
```

Comment:
This change changes the hash of this file, which then causes to fail the `./run_tests.py`. To fix that, just do:
```
./run_test.py -u
```
To regenerate the tests and commit the changes.

Reply:
I have updated the reference files using `python run_tests.py -u` which modified existing files and created new files as well. Didn't expect the updated files to be these many in number. Could you kindly check if this is as expected !?

Reply:
Done. Whenever you face this kind of situation in LPython follow the steps below (assuming you are under `lpython`'s root),

1. `cd tests/reference`.
2. `rm ./*` (execute this command very carefully, because it deletes many files).
3. `cd ../../`
4. `python run_tests.py -u`
5. `git add .`
6. `git commit -m "Updated reference tests"`.

Reply:
Thanks for the assist. I will keep this in mind.

Reply:
> 2\. rm ./*

I want to run this, but I guess the command is for a linux terminal. I'll have to figure out it's windows equivalent.